models/product/src/builder.spec.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TProduct, TProductGraphql } from './types';
+import * as Product from '.';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TProduct, TProduct>(
+      'default',
+      Product.random(),
+      expect.objectContaining({
+        id: expect.any(String),
```

Comment:
This one would have the `skus` array, correct?

Reply:
⚡